docs/guides/run-capability.md

```diff
@@ -215,52 +215,40 @@ async function main() {
 main();
 ```
 
-:::info
-Replace `scope/profile-name`, `UseCaseName` and inputs for `.perform` method with the use case details you actually want to use.
+_Replace `scope/profile-name`, `UseCaseName` and inputs for `.perform` method with the use case details you actually want to use._
 
+:::info
 For details on SuperfaceClient API, please consult [OneSDK reference](/reference/one-sdk-js).
 :::
 
 ## Run the app
 
-You can then run your app that performs the use case.
+Now run the application to perform the use case and check the results:
 
 ```shell
 node app.js
 ```
 
 :::tip Offline Use
-To use capibility without the use of Superface [remote registry](https://superface.ai/catalog). You have to import capabilities into project and ensure `super.json` has valid paths to your capabilities.
+To use capability without the use of Superface [remote registry](https://superface.ai/catalog). You have to import capabilities into project and ensure `super.json` has valid paths to your capabilities.
 
-TODO: link to offline use guide
+<!-- TODO: link to offline use guide -->
 :::
 
-### DEBUG
-
-Superface uses [DEBUG package](https://github.com/visionmedia/debug), which you can use when running or testing your app. To use it, you simply have to add environment variable `DEBUG="superface*"` before running tests.
-
-There are mutiple parts of Superface, which implemented this package and created debug context:
-
-in `OneSDK`:
+### Observe and debug API calls
 
-- `superface:config`
-- `superface:profile-parameter-validator`
-- `superface:map-interpreter`
-- `superface:http`, `superface:http:sensitive`
-- `superface:superjson`
-- `superface:failover`, `superface:failover:sensitive`
-- `superface:bound-profile-provider`
-- `superface:registry`
-- `superface:usecase`
-- `superface:lib/env`
-- `superface:events`
-- `superface:metric-reporter`
+OneSDK uses the [debug package](https://github.com/visionmedia/debug) which is useful for observing the behavior of the SDK and debugging. To use it, set environment variable to `DEBUG="superface*"` before running the application:
 
-in `Parser`:
+```shell
+DEBUG="superface*" node app.js
+```
 
-- `superface-parser:map-validator`
-- `superface-parser:profile-io-analyzer`
+To observe API calls, you can use `superface:http` debug context.
 
-To know more about setting up specific contexts, check out [conventions](https://github.com/visionmedia/debug#conventions) and [wildcards](https://github.com/visionmedia/debug#wildcards) of debug package.
+:::info Credentials
+If you want to debug sensitive information, you can use context `superface:http:sensitive`
+:::
```

Comment:
I recommend tuning this admonition around like: by default with * this includes sensitive data.

```
:::caution Sensitive Output

Using the `superface:http*` context will output full HTTP requests and responses including API credentials and other sensitive data. 

:::
```